visidata/features/ping.py

```diff
@@ -145,7 +151,7 @@ def iterload(self):
 vd.option('ping_interval', 0.1, 'wait between ping rounds, in seconds', sheettype=PingSheet)
 
 PingSheet.options.null_value = False
-BaseSheet.addCommand('', 'open-ping', 'vd.push(openSource(input("ping: ", type="hostip"), filetype="ping"))', 'open sheet to ping input IP Address')
+BaseSheet.addCommand('', 'open-ping', 'open_ping()', 'open sheet to ping input IP Address')
```

Comment:
Please include at least `push` and `input` strings in the execstr; this is how the e.g. menu system knows that it takes input and pushes a new sheet.  You can make an `inputIP` if that is helpful; and move the `open_ping` to the VisiData object and return a Sheet (like other loaders) so that someone can do `vd -f ping 4.4.4.4`.

Reply:
Oh, I didn't realize the strings needed to be in the execstr. I've put them back.

I kept `new_ping` that handled `vd -f ping 4.4.4.4`. And I made `open_ping` part of the VisiData object as you suggested, since it's called if the ping target happens to exist as a local file.

I modeled `makePingStats` on the `freqtbl` call to `makeFreqTable`. Please feel free to make any changes if that's not a good fit for current vd style.